### PR TITLE
Importing Audio Init Params

### PR DESCRIPTION
**Pull request by engineer124**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

This imports initialization parameters used by Audio Heap while initializing audio. This file is similar to OoTs:
https://github.com/zeldaret/oot/blob/master/src/code/audio_init_params.c

For direct comparison:
- heapSize: MM 0x137F00 <--> 0x37F00 OoT
- initPoolSize: MM 0x1C480 <--> 0xE0E0 OoT
- permanentPoolSize: MM 0x1A000 <--> 0xBCE0 OoT

(gTatumsPerBeat is identical between the two games).

So in other words, audio heap in MM got leveled up.

### CODE REVIEW

**Comment by isghj5** [comment]
`For direct comparison:

    heapSize: MM 0x137F00 <--> 0x37F00 OoT
    initPoolSize: MM 0x1C480 <--> 0xE0E0 OoT
    permanentPoolSize: MM 0x1A000 <--> 0xBCE0 OoT
`
Cool, although if this was me I would add these three var names next to their data on line audio_init_parms::8

**Review by isghj5 — APPROVED** [review]

File: tools/disasm/variables.txt [hunk 1]
```diff
@@ -3879,11 +3879,8 @@
     0x801E1070:("jtbl_801E1070","UNK_PTR","",0x4),
     0x801E10B0:("jtbl_801E10B0","UNK_PTR","",0x4),
     0x801E10C4:("jtbl_801E10C4","UNK_PTR","",0x4),
-    0x801E1100:("D_801E1100","UNK_TYPE2","",0x2),
-    0x801E1102:("D_801E1102","UNK_TYPE2","",0x2),
-    0x801E1104:("D_801E1104","UNK_TYPE4","",0x4),
-    0x801E1108:("D_801E1108","UNK_TYPE4","",0x4),
-    0x801E110C:("D_801E110C","UNK_TYPE4","",0x4),
+    0x801E1100:("D_801E1100","UNK_TYPE4","",0x4),
```
Comment (AngheloAlf) [inline tools/disasm/variables.txt:3882]:
```suggestion
    0x801E1100:("D_801E1100","s16","",0x4),
```

**Review by AngheloAlf — APPROVED** [review]
Just a small nitpick, but I'll approve anyways in the meanwhile

**Review by EllipticEllipsis — APPROVED** [review]

File: src/code/audio/audio_init_params.c [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#include "global.h"
+
+const s16 D_801E1100[] = {
```
Comment (Kenix3) [inline src/code/audio/audio_init_params.c:3]:
```suggestion
const s16 gAudioTatumInit[] = {
```
maybe?

File: src/code/audio/audio_init_params.c [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+    0x1C00, // unused
+    0x30,   // gTatumsPerBeat
+};
+
+const AudioContextInitSizes D_801E1104 = {
```
Comment (Kenix3) [inline src/code/audio/audio_init_params.c:8]:
```suggestion
const AudioContextInitSizes gAudioContextInitSizes = {
```

**Review by Kenix3 — CHANGES_REQUESTED** [review]

**Review by Kenix3 — APPROVED** [review]
